prac_07/myguitars.py

```diff
@@ -0,0 +1,68 @@
+from guitar import Guitar
```

Comment:
Missing program docstring - this is easy to miss.

Reply:
I think I manage to do this at least once per week. I'll try make sure to check before next weeks review.